draft-ietf-jsonpath-base.md

```diff
@@ -371,45 +373,45 @@ A JSONPath expression may use a combination of bracket and dot notations.
 
 Dot notation is a shorthand way of writing certain bracket notations.
 
-This document treats the bracket notations as canonical and defines various shorthands in terms
+This document treats bracket notation as canonical and defines various shorthands in terms
 of bracket notation. Examples and descriptions use shorthands where convenient.
 
 ### Selectors
 
-A wildcard `*` ({{wildcard}}) in the expression `[*]` selects all children of an
-object or an array and in the expression `..[*]` selects all descendants of an object or an array.
+A wildcard `*` ({{wildcard}}) in the expression `[*]` selects all children of a
+value and in the expression `..[*]` selects all descendants of a value.
```

Comment:
Can we use "node" here?
"value" is seriously confusing.

Reply:
Sure.

Reply:
Done.